app/assets/js/rtMedia.js

```diff
@@ -568,6 +595,41 @@ jQuery( 'document' ).ready( function( $ ) {
 		} );
 	}
 
+	/**
+	 * Sets Cookie.
+	 * 
+	 * @param {string} cname
+	 * @param {string} cvalue
+	 * @param {int} exdays
+	 * @return void
+	 */
+	function setCookie(cname, cvalue, exdays) {
+	    var d = new Date();
+	    d.setTime(d.getTime() + (exdays * 24 * 60 * 60 * 1000));
+	    var expires = "expires="+d.toUTCString();
```

Comment:
could be
`d.setTime( d.getTime() + ( exdays * 24 * 60 * 60 * 1000 ) );`

`var expires = "expires=" + d.toUTCString();`